### PR TITLE
2player splitscreen

### PR DESCRIPTION
**Pull request by Raptorq1**
Check the scene file named node2d that's split screen

### CODE REVIEW

File: Savedata.bin [hunk 1]
```diff
@@ -0,0 +1 @@
+{"Emerald":0,"lives":3,"player_selected":1}
```
Comment (DimensionWarped) [inline Savedata.bin:1]: PRs need to only include changes relevant to the specific change.

File: basezone.tscn [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[gd_scene format=3 uid="uid://bwbosgjfdtxxk"]
+
+[node name="Basezone" type="Label"]
+offset_left = 86.0
+offset_top = 17.0
+offset_right = 206.0
+offset_bottom = 28.0
+text = "Base Zone Act 1"
+uppercase = true
```
Comment (DimensionWarped) [inline basezone.tscn:9]: PRs need to only include changes relevant to the specific change.

File: basezone_2.tscn [hunk 3]
```diff
@@ -0,0 +1,9 @@
+[gd_scene format=3 uid="uid://38s3gv258vpr"]
+
+[node name="basezone2" type="Label"]
+offset_left = 85.0
+offset_top = 30.0
+offset_right = 205.0
+offset_bottom = 41.0
+text = "Base Zone Act 1"
+uppercase = true
```
Comment (DimensionWarped) [inline basezone_2.tscn:9]: PRs need to only include changes relevant to the specific change.

File: canvas_layer.tscn [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+rotation = -1.5708
+texture_normal = ExtResource("2_jqsmy")
+texture_pressed = ExtResource("3_k43sn")
+action = "gm_left"
+
+[node name="TouchScreenButton5" type="TouchScreenButton" parent="."]
+position = Vector2(337, 185)
+texture_normal = ExtResource("4_bmsfn")
+texture_pressed = ExtResource("5_p7ic5")
+action = "gm_action"
```
Comment (DimensionWarped) [inline canvas_layer.tscn:46]: PRs need to only include changes relevant to the specific change.

And I'm pretty sure we have no intention of exposing touch screen controls in the base game (at least unless it's a toggleable feature, but that should be a separate PR)

File: character_origin.tscn [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+position = Vector2(-5, 3)
+texture = ExtResource("4_8wwib")
+region_enabled = true
+region_rect = Rect2(488, 8, 32, 40)
+
+[node name="CharacterAnimations" type="AnimationPlayer" parent="."]
+autoplay = "Animate"
+libraries = {
+"": SubResource("AnimationLibrary_wio08")
+}
```
Comment (DimensionWarped) [inline character_origin.tscn:186]: PRs need to only include changes relevant to the specific change.

File: fadeOut.res
Comment (DimensionWarped) [inline fadeOut.res:1]: PRs need to only include changes relevant to the specific change.

File: node_2d.tscn
Comment (DimensionWarped) [inline node_2d.tscn:1]: This file being named node_2d is a problem.

File: project.godot [hunk 6]
```diff
@@ -198,6 +212,13 @@ ui_reset={
 "events": [Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":-1,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":4194333,"physical_keycode":0,"key_label":0,"unicode":0,"echo":false,"script":null)
 ]
 }
+gm_start={
+"deadzone": 0.5,
+"events": [Object(InputEventMouseButton,"resource_local_to_scene":false,"resource_name":"","device":-1,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"button_mask":0,"position":Vector2(0, 0),"global_position":Vector2(0, 0),"factor":1.0,"button_index":1,"canceled":false,"pressed":false,"double_click":false,"script":null)
+, Object(InputEventJoypadButton,"resource_local_to_scene":false,"resource_name":"","device":-1,"button_index":6,"pressure":0.0,"pressed":false,"script":null)
+, Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":4194309,"key_label":0,"unicode":0,"echo":false,"script":null)
+]
+}
```
Comment (DimensionWarped) [inline project.godot:221]: PRs need to only include changes relevant to the specific change.

**Review by DimensionWarped — CHANGES_REQUESTED** [review]
I might look into creating a separate PR out of this later, but this can't be merged as is.

**Comment by Raptorq1** [comment]
Okay
